components/Modal.js

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Modal from "@mui/material/Modal";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
```

Comment:
@VitorLelis started using Ant Design in other parts of the project. I think we should only have one components library. We can choose `mui` (it would require that you rewrite those "other parts" using it), but I've never used it, what are its pros?